### PR TITLE
Fixes #79 x64 bit setup

### LINKED ISSUE #79: x64 bit installer version of Matlab toolbox

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 environment:
-  app_version: '7.3.0'
+  app_version: '7.4.0'
```
Comment (msevestre) [inline appveyor.yml:2]: new version

File: packages.config [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  	<package id="OSPSuite.FuncParser" version="3.0.0.6"/>
-  	<package id="OSPSuite.SimModelSolver_CVODES282" version="3.0.0.7" />
-	<package id="OSPSuite.SimModel" version="3.0.0.16"/>
-	<package id="OSPSuite.SimModelComp" version="3.0.0.16"/>
+  	<package id="OSPSuite.FuncParser" version="3.0.1.9"/>
```
Comment (msevestre) [inline packages.config:3]: packages used in OSPSuite.Core

File: rakefile [hunk 3]
```diff
@@ -89,14 +89,16 @@ def copy_setup_files
 end
 
 def copy_to_lib_dir(target_lib_dir) 
+  arch = 'x64'
   copy_depdencies lib_dir, target_lib_dir do
-    copy_files File.join('DCI','x86'), 'dll'
+    copy_files File.join('DCI', arch), 'dll'
     copy_schema_xsd
     copy_dcimatlab_dll
+    copy_dcimatlab_mexw64
```
Comment (msevestre) [inline rakefile:97]: added that guy as well

File: setup/setup.wxs [hunk 4]
```diff
@@ -13,12 +13,6 @@
         <Property Id="ALLUSERS" Value="1"/>
         <Media Id="1" Cabinet="media1.cab" EmbedCab="yes"/>
 
-        <?if $(sys.BUILDARCH) = x64 ?>
```
Comment (msevestre) [inline setup/setup.wxs:16]: not required

**Review by msevestre — COMMENTED** [review]
if ml_readtab and ml_writetab need to be migrated, this will have to be done here

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**